hugo/content/docs/delete-a-team.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Deleting a team in Fluidkeys"
-description: "Fluidkeys automates key rotation and exchange amongst a team. This guide will get you setup as a team and each member of the team setup with new PGP keys."
+weight: 3
```

Comment:
interesting, what are these `weight` variables? I don't see any reference to them?

Reply:
they are used to determine where they appear in the `{{ range .Pages }}` ... `{{ end }}` loop in `section.html`.